### ISSUE TITLE
Problem with datastore name?

### ISSUE THREAD

**Issue by sasgithub** [pbertera/pmvcb#1]
I'm running into issues 
# pmvcb -v Ubuntu_12.04.clone -d /vmfs/volumes/50a26bf9-b97d48fe-1d61-003048ff7069/backups -h thinkmate1 -s '' -q

INFO: [Ubuntu_12.04.clone] VMFS Volume: datastore1 (1)
INFO: [Ubuntu_12.04.clone] VMX: Ubuntu_12.04.clone/ls.vmx
INFO: [Ubuntu_12.04.clone] VMX Path: /vmfs/volumes/datastore1 (1)/Ubuntu_12.04.clone/ls.vmx
INFO: [Ubuntu_12.04.clone] VMX Dir: /vmfs/volumes/datastore1 (1)/Ubuntu_12.04.clone
INFO: [Ubuntu_12.04.clone] VM ID: 32
ash: syntax error: "(" unexpected
ERROR: [Ubuntu_12.04.clone] cannot copy vmx in local
Create Snapshot:
INFO: [Ubuntu_12.04.clone] Snapshot PMVCB-Ubuntu_12.04.clone created
INFO: [Ubuntu_12.04.clone] Saving VMX
Remove Snapshot:
INFO: [Ubuntu_12.04.clone] Snapshot consolidated

I assume the problem is the parentheses in the datastore name.

Has anyone else run into this?

**Comment by pbertera** [comment]
Yes, you're right: I guess that parentheses in datastore name cause the issue on scp execution.
